From 8a034cdf1227700f964bb4bcc64d05400a04f66a Mon Sep 17 00:00:00 2001
From: wwidjaja <w.widjaja@intel.com>
Date: Tue, 12 May 2020 13:20:13 +0800
Subject: [PATCH] PONRTSYS-6644 - keep UART0 clock on

  ignore Clock for UART0 even if no UART0 driver using it,
  else reset (GLOBAL_SW_RESET) will fail.
---
 drivers/clk/intel/clk-prx300.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/intel/clk-prx300.c b/drivers/clk/intel/clk-prx300.c
index 14d44f2760b7..8f0a3c0df16a 100644
--- a/drivers/clk/intel/clk-prx300.c
+++ b/drivers/clk/intel/clk-prx300.c
@@ -174,8 +174,8 @@ static const struct intel_clk_branch prx300_branch_clks[] __initconst = {
 		   13, GATE_CLK_HW, 0),
 	INTEL_GATE(PRX300_GCLK_GPTC2, "g_gptc2", NULL, 0, CGU_GATE1,
 		   14, GATE_CLK_HW, 0),
-	INTEL_GATE(PRX300_GCLK_URT0, "g_uart0", NULL, 0, CGU_GATE1,
-		   17, GATE_CLK_HW, 0),
+	INTEL_GATE(PRX300_GCLK_URT0, "g_uart0", NULL, CLK_IGNORE_UNUSED,
+		   CGU_GATE1, 17, GATE_CLK_HW, 0),
 	INTEL_GATE(PRX300_GCLK_URT1, "g_uart1", NULL, 0, CGU_GATE1,
 		   18, GATE_CLK_HW, 0),
 	INTEL_GATE(PRX300_GCLK_SECPT, "g_secpt", NULL, CLK_IGNORE_UNUSED,
